.gitignore

```diff
@@ -114,3 +114,4 @@ docbuild/
 
 # generated perf test scripts
 test/perf/scripts/**
+.idea
```

Comment:
nit: can you make this a new section with a comment on what .idea files are? I presume they're related to your IDE in some way?

Reply:
Yes, it is related to the IDE I use. should I remove it altogether and just keep it in my local instead?

Reply:
I imagine we will acquire these over time anyway. I think a comment line above to note that this is to ignore files created by IDEA would be fine. We already have things for VS and VS Code, after all.

Reply:
Done 👍